frontend/src/app/utils/smartyStreets.ts

```diff
@@ -55,3 +55,34 @@ export const getBestSuggestion = async (
     console.error("Unable to validate address:", error.message);
   }
 };
+
+function toLowerStripWhitespace(s: string | null): string {
+  if (s === null) {
+    return "";
+  }
+  return s.toLocaleLowerCase().replace(/\s/, "");
+}
+
+export function suggestionIsCloseEnough(
+  original: Address,
+  suggested: Address | undefined
+): boolean {
+  if (typeof suggested === "undefined") {
+    return false;
+  }
+
+  const fields: (keyof Address)[] = ["city", "state", "street", "streetTwo"];
+  fields.forEach((field) => {
+    if (
+      toLowerStripWhitespace(original[field]) !==
+      toLowerStripWhitespace(suggested[field])
+    ) {
+      return false;
```

Comment:
`forEach` doesn't do anything with the return value, does it? I think the intent here was a `for` loop

Reply:
A `some` or `every` loop might work here as well, but @jeskew-gov is correctly pointing out that the `forEach` loop is currently not affecting the return value of the `suggestionIsCloseEnough` function

Reply:
Oh, yeah, duh. Thanks!

Fixed in 93e7b241